### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -17,55 +17,55 @@
...
+    <string name="clear_button">Clear text button</string>
+    <string name="overflow_menu_button">Overflow menu button</string>
     <string name="search_view_hint">Type something…</string>
     <string name="title_activity_settings">Settings</string>
     <string name="settings_category_notification_category">Notifications</string>
-    <string name="settings_enable_notification_title">Show shortcut in notification panel</string>
-    <string name="settings_landscape_90">90° orientation navigation bar position</string>
-    <string name="settings_landscape_270">270° orientation navigation bar position</string>
+    <string name="settings_enable_notification_title">Shortcut in notification panel</string>
+    <string name="settings_landscape_90">90° shifted navigation bar</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:28]: Is this string used anywhere?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -17,55 +17,55 @@
...
+    <string name="settings_landscape_270">270° shifted navigation bar</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:29]: Is this string used anywhere?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (avuton) [reply app/src/main/res/values/strings.xml:28]: [yes](https://github.com/KeikaiLauncher/KeikaiLauncher/blob/117e515f77c309c145d3f069010f640f7024c44e/app/src/main/res/xml/preferences.xml#L54).

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (avuton) [reply app/src/main/res/values/strings.xml:29]: [yes](https://github.com/KeikaiLauncher/KeikaiLauncher/blob/117e515f77c309c145d3f069010f640f7024c44e/app/src/main/res/xml/preferences.xml#L60).

**Comment by comradekingu** [comment]
@avuton Yeah…

**Comment by avuton** [comment]
All open PRs were closed due to the branch change, and there's nothing actually wrong with that. With that said, this PR has been open too long with no action anyway. It was nothing personal, but I didn't see any issue in how the wording was written. I'm happy to discuss this further if you'd like to; I'm always happy to hear someone argue a point.

**Pull request closed without merging**